apps/scripts/gpc_compile_link.sh

```diff
@@ -8,14 +8,14 @@ if [ "$1" == "-help" ];then
     cat ./README.txt
     exit 1
 fi
-str=riscv/install/rv32e/bin/
-mod=e
+str=riscv/install/rv32i/bin/
+mod=i
 if [ "$1" == "-e" ];then
     str=riscv_e/install/rv32e/bin/
     mod=e
 fi
 
-GNU_DIR=/home/saar/projects/$str
+GNU_DIR=/home/adlv/projects/$str
```

Comment:
You really shouldn't have your private path in scripts that go to everyone. You can replace it with, for example:
```bash
GNU_DIR=$GNU_DIR_PREFIX/projects/$str
```
And then set this variable in your shell before you start the scripts